packages/react-instantsearch-core/src/components/Index.tsx

```diff
@@ -29,7 +31,6 @@ type State = {
  * @kind widget
  * @name <Index>
  * @propType {string} indexName - index in which to search.
```

Comment:
They are outdated + the file is written in TypeScript. It's probably better to remove them.